api/spec/src/customer.tsp

```diff
@@ -1,32 +1,103 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+import "./errors.tsp";
 import "./billing.tsp";
```

Comment:
Please remove the billing providers from the customer object.

Reply:
Done